page/ajax/working-with-jsonp.md

```diff
@@ -31,4 +31,8 @@ $.ajax({
 });
 ```
 
+If you'd like to access community tables you have to add `env: "http://datatables.org/alltables.env"` to the data object otherwise you would get an error message with `No definition found for Table`.
```

Comment:
Is there some way we can not go through this all and keep it simple instead? I can imagine this is already too complicated for many less-experienced people.